src/error/handleValidationError.js

```diff
@@ -0,0 +1,15 @@
+import { StatusCodes } from 'http-status-codes'
+
+function handleValidationError (err, req, res, next) {
+  if (err && err.error && err.error.isJoi) {
+    const errorMessages = err.error.details.map(details => {
```

Comment:
Probably good format of error messages for `development` mode but not for 'production' and real `users` :) But I think for current task it's enough

Reply:
I thought Joi has good description of validation error and it can be used for prod